### PR TITLE
migrated KB's to Fundamentals

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+pcx-content-type: concept
+---
+
+# Cloudflare's Cookies
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/cloudflare-cookies.md]:
```suggestion
# Cloudflare Cookies
```
suggesting this for consistency with the following header

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Cloudflare uses various cookies to maximize network resources, manage traffic, and protect our customers’ sites from malicious traffic.
+
+## Understanding the Cloudflare Cookies
+
+As defined in our [Privacy Policy](https://www.cloudflare.com/privacypolicy/), the \__cflb, \__cf_bm, \__cf_ob_info and \__cf_use_ob, and \__cfwaitingroom cookies are strictly necessary to provide the services requested by our customers.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/cloudflare-cookies.md]: should we highlight cookie names in any way? my gut would suggest using backticks (so `\__cflb`) but I see the former KB uses italics

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+A mentioned in our Privacy Policy, Cloudflare encourages our customers to disclose the use of these cookies to their end users and, in some jurisdictions, customers may be required to do it by law.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/cloudflare-cookies.md]:
```suggestion
As mentioned in our Privacy Policy, Cloudflare encourages our customers to disclose the use of these cookies to their end users and, in some jurisdictions, customers may be required to do it by law.
```
Also, we may consider using clearer verbiage here: `do it by law`. I think I know what you mean but I'm not 100% sure what `it` refers to, and when law is involved it's always best to be extra explicit

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+Cookie data is processed in Cloudflare's data center in the United States and is subject to the cross-border data transfer section 7 of the Cloudflare [Privacy Policy](https://www.cloudflare.com/privacypolicy/).
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/cloudflare-cookies.md]: This makes me wonder which data center, since we have [so many](https://www.cloudflare.com/en-gb/network/). is this something we can clarify?

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+### \*__cflb* cookie for Cloudflare Load Balancer session affinity
+
+When enabling session affinity with [Cloudflare Load Balancer](https://developers.cloudflare.com/load-balancing/understand-basics/session-affinity), Cloudflare sets a \__cflb cookie with a unique value on the first response to the requesting client. Cloudflare routes future requests to the same origin, optimizing network resource usage. In the event of a failover, Cloudflare sets a new \__cflb cookie to direct future requests to the failover pool.
+
+The \__cflb cookie allows us to return an end user to the same customer origin for a specific period of time (customer-configured), which in turn allows the customer origin to maintain an end user’s experience seamlessly (for example, keeping an end user’s items in a shopping cart while they continue to navigate around the website). This cookie is a session cookie that lasts from several seconds up to 24 hours.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/cloudflare-cookies.md]:
```suggestion
The \__cflb cookie allows Cloudflare to return an end user to the same customer origin for a specific period of time (customer-configured). This allows the end user to have a seamless experience (for example, this cookie is used for keeping an end user’s items in a shopping cart while they continue to navigate around the website). This cookie is a session cookie that lasts from several seconds up to 24 hours.
```
1. What does customer-configured refer to? is it the specific period of time? if so, I would rephrase as `for a specific period of time configured by the customer.`
2. I have attempted to break down the following sentence to improve readability — see if the rephrasing makes sense!

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+<Aside type="note">
+
+Currently Cloudflare only supports Session Affinity in "orange-cloud" (proxied) mode.
+
+</Aside>
+
+### \*__cf_bm* cookie for Cloudflare bot products
+
+Cloudflare's [bot products](https://developers.cloudflare.com/bots/) identify and mitigate automated traffic to protect your site from bad bots. Cloudflare places the \__cf_bm cookie on end user devices that access customer sites that are protected by Bot Management or Bot Fight Mode. The \__cf_bm cookie is necessary for the proper functioning of these bot solutions.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/cloudflare-cookies.md]:
```suggestion
Cloudflare's [bot products](https://developers.cloudflare.com/bots/) identify and mitigate automated traffic to protect your site from bad bots. Cloudflare places the \__cf_bm cookie on end-user devices that access customer sites protected by Bot Management or Bot Fight Mode. The \__cf_bm cookie is necessary for these bot solutions to function properly.
```

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+This cookie expires after 30 minutes of continuous inactivity by the end user. The cookie contains information related to the calculation of Cloudflare’s proprietary bot score and, when Anomaly Detection is enabled on Bot Management, a session identifier. The information in the cookie (other than time-related information) is encrypted and can only be decrypted by Cloudflare.
+
+A separate \__cf_bm cookie is generated for each site that an end user visits, as Cloudflare does not follow users from site to site or from session to session by merging various \__cf_bm identifiers into a profile. The \__cf_bm cookie is generated independently by Cloudflare, and does not correspond to any user ID or other identifiers in a customer’s web application.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/cloudflare-cookies.md]:
```suggestion
A separate \__cf_bm cookie is generated for each site that an end user visits, as Cloudflare does not follow users from site to site or from session to session by merging various \__cf_bm identifiers into a profile. The \__cf_bm cookie is generated independently by Cloudflare, and does not correspond to any user ID or other identifiers in a customer’s web application.
```
`Cloudflare does not follow users from site to site or from session to session by merging various \__cf_bm identifiers into a profile` This sentence is a bit unclear to me — does Cloudflare merge the identifiers or not? See what I mean? (also I would probably use a verb like `track`)

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+
+[Cloudflare’s Waiting Room](https://developers.cloudflare.com/waiting-room/) product enables a waiting room for a particular host and path combination within a zone. Visitors are put in the waiting room and provided an estimate of when they will be allowed to access the application, if not immediately available.
+
+The \_cfwaitingroom cookie is only used to track visitors that access a Waiting Room enabled host and path combination for a zone. Visitors using a browser that does not accept cookies cannot visit the host and path combination while the Waiting Room is active. For more details, see [Waiting Room cookie](https://developers.cloudflare.com/waiting-room/reference/waiting-room-cookie).
+
+### Additional cookies used by the Challenge Platform
+
+The table below shows additional cookies used by the Challenge Platform.
+
+
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/cloudflare-cookies.md]: remove extra line

File: products/fundamentals/src/content/get-started/how-cloudflare-works.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
+---
+order: 1
+pcx-content-type: how-to
+---
+
+# How Cloudflare works
+
+More than just [Content Delivery (CDN)](https://www.cloudflare.com/learning/cdn/what-is-a-cdn/) services, customers rely on Cloudflare’s global network to enhance security, performance, and reliability of anything connected to the Internet.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/how-cloudflare-works.md]: I'm not sure I understand this opening — what do we want to say here? Is it that Cloudflare is more than a CDN service? Or that users rely on more than the CDN services we offer?

File: products/fundamentals/src/content/get-started/how-cloudflare-works.md [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+## Security
+
+Cloudflare stops malicious traffic before it reaches your origin web server. Cloudflare analyzes potential threats in visitor requests based on a number of characteristics:
+
+- Visitor's IP address
+- Resources requested
+- Request payload and frequency
+- Customer-defined firewall rules
+
+A DNS lookup of a proxied (orange-clouded) Cloudflare subdomain returns [Cloudflare IP addresses](https://www.cloudflare.com/ips/). Proxied traffic comes to our edge and then we forward the request to your server. Cloudflare masks your origin IP address for proxied DNS records so attackers cannot bypass Cloudflare and directly attack your origin web server.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/how-cloudflare-works.md]:
```suggestion
A DNS lookup of a proxied (orange-clouded) Cloudflare subdomain returns [Cloudflare IP addresses](https://www.cloudflare.com/ips/). Proxied traffic comes to Cloudflare's edge and then Cloudflare forwards the request to your server. Cloudflare masks your origin IP address for proxied DNS records so attackers cannot bypass Cloudflare and directly attack your origin web server.
```

File: products/fundamentals/src/content/get-started/how-cloudflare-works.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+
+Visitor <--[Connection 1]--> Cloudflare Edge <--[Connection 2]--> Origin Server
+
+A DNS lookup of an unproxied (grey-clouded) Cloudflare subdomain returns the IP address that you've entered for the record. Unproxied traffic goes directly to your origin server and does not receive any of the benefits of using Cloudflare.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/how-cloudflare-works.md]:
```suggestion
A DNS lookup of an unproxied (grey-clouded) Cloudflare subdomain returns the IP address that you have entered for the record. Unproxied traffic goes directly to your origin server and does not receive any of the benefits of using Cloudflare.
```

File: products/fundamentals/src/content/get-started/how-cloudflare-works.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+[Create your Cloudflare account and add a domain](https://support.cloudflare.com/hc/en-us/articles/201720164) to review our security benefits.
+
+## Performance
+
+Cloudflare optimizes the delivery of website resources for your visitors. Cloudflare’s data centers serve your website’s [static resources](https://www.cloudflare.com/learning/cdn/caching-static-and-dynamic-content/) and ask your origin web server for dynamic content. [Cloudflare’s global network](https://www.cloudflare.com/network/) provides a faster route from your site visitors to our data centers than would be available to a visitor directly requesting your site. Even with Cloudflare between your website and your visitors, resource requests arrive to your visitor sooner.
+
+## Reliability
+
+Cloudflare’s globally distributed [any cast network](https://www.cloudflare.com/learning/cdn/glossary/anycast-network/) routes visitor requests to the nearest Cloudflare data center.  Cloudflare distributed DNS responds to website visitors with Cloudflare IP addresses for traffic you proxy to Cloudflare.  This also provides security by hiding the specific IP address of your origin web server.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/how-cloudflare-works.md]:
```suggestion
Cloudflare’s globally distributed [Anycast network](https://www.cloudflare.com/learning/cdn/glossary/anycast-network/) routes visitor requests to the nearest Cloudflare data center. Cloudflare distributed DNS responds to website visitors with Cloudflare IP addresses for traffic you proxy to Cloudflare. This also provides security by hiding the specific IP address of your origin web server.
```
Capitalization suggested for consistency with content in the link provided ^

File: products/fundamentals/src/content/get-started/how-cloudflare-works.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+
+<Aside type="note">
+
+Cloudflare-proxied domains share IP addresses from a pool that belongs to the Cloudflare network. As a result, Cloudflare does not offer dedicated or exclusive IP addresses. To reduce the number of Cloudflare IPs that your domain shares with other Cloudflare customer domains, upgrade to a [Business or Enterprise plan and upload a Custom SSL certificate](https://developers.cloudflare.com/ssl/edge-certificates/custom-certificates).
+
+</Aside>
+
+Also, our flat-rate pricing structure provides predictability and reliability in your [CDN](https://www.cloudflare.com/cdn-y/) and [DDoS](https://www.cloudflare.com/ddos/) bandwidth expenses.  Cloudflare does not have bandwidth limits for domains on the Free, Pro and Business plans as long as those domains comply with our [Terms of Service](https://www.cloudflare.com/terms/). However, your hosting provider may still impose limits on bandwidth usage and/or charge for bandwidth.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/how-cloudflare-works.md]:
```suggestion
Also, our flat-rate pricing structure provides predictability and reliability in your [CDN](https://www.cloudflare.com/cdn-y/) and [DDoS](https://www.cloudflare.com/ddos/) bandwidth expenses. Cloudflare does not have bandwidth limits for domains on the Free, Pro and Business plans as long as those domains comply with our [Terms of Service](https://www.cloudflare.com/terms/). However, your hosting provider may still impose limits on bandwidth usage and/or charge for bandwidth. 
```

File: products/fundamentals/src/content/get-started/how-cloudflare-works.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+
+Cloudflare-proxied domains share IP addresses from a pool that belongs to the Cloudflare network. As a result, Cloudflare does not offer dedicated or exclusive IP addresses. To reduce the number of Cloudflare IPs that your domain shares with other Cloudflare customer domains, upgrade to a [Business or Enterprise plan and upload a Custom SSL certificate](https://developers.cloudflare.com/ssl/edge-certificates/custom-certificates).
+
+</Aside>
+
+Also, our flat-rate pricing structure provides predictability and reliability in your [CDN](https://www.cloudflare.com/cdn-y/) and [DDoS](https://www.cloudflare.com/ddos/) bandwidth expenses.  Cloudflare does not have bandwidth limits for domains on the Free, Pro and Business plans as long as those domains comply with our [Terms of Service](https://www.cloudflare.com/terms/). However, your hosting provider may still impose limits on bandwidth usage and/or charge for bandwidth. 
+
+Note that additional terms may apply to:
+
+- Self-serve domains with add-on features (contact  for additional details)
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/how-cloudflare-works.md]: content in parentheses seems incomplete?

File: products/fundamentals/src/content/get-started/how-cloudflare-works.md [hunk 15]
```diff
@@ -0,0 +1,58 @@
...
+- Domains in the Enterprise plan (contact your Cloudflare Account Team for further information)
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/how-cloudflare-works.md]: these two parenthetical sentences seem to have the same intent — should we uniform them? e.g., always say `for further information` vs. `for additional details`

File: products/fundamentals/src/content/get-started/http-request-headers.md [hunk 16]
```diff
@@ -0,0 +1,106 @@
+---
+pcx-content-type: concept
+---
+
+# HTTP Request headers
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/http-request-headers.md]:
```suggestion
# HTTP request headers
```

File: products/fundamentals/src/content/get-started/http-request-headers.md [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+Add the [CF-Ray header to your origin web server logs](https://support.cloudflare.com/hc/articles/203118044#h_f7a7396f-ec41-4c52-abf5-a110cadaca7c) to match requests proxied to Cloudflare to requests in your server logs. Enterprise customers can also see all requests via [Cloudflare Logs](https://support.cloudflare.com/hc/en-us/articles/216672448-Enterprise-Log-Share-REST-API).
+
+</div>
+</details>
+
+<details>
+<summary>CF-IPCountry</summary>
+<div>
+
+CF-IPCountry contains a two character country code of the originating visitor’s country. XX is used for unknown country information. This header is added to requests by enabling [Cloudflare IP Geolocation](https://support.cloudflare.com/hc/en-us/articles/200168236-What-does-Cloudflare-IP-Geolocation-do-) in the dashboard. For example: ``CF-IPCountry: US``.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/http-request-headers.md]: Should we consider highlighting these headers too? e.g. between `` ?

File: products/fundamentals/src/content/get-started/http-request-headers.md [hunk 18]
```diff
@@ -0,0 +1,106 @@
...
+</Aside>
+
+</div>
+</details>
+
+<details>
+<summary>X-Forwarded-For</summary>
+<div>
+
+X-Forwarded-For maintains proxy server and original visitor IP addresses. If there was no existing X-Forwarded-For header in the request sent to Cloudflare, X-Forwarded-For has an identical value to the CF-Connecting-IP header. For example: ``X-Forwarded-For: 203.0.113.1``.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/http-request-headers.md]: curious — why the double back ticks? do we just want to highlight this as code?

File: products/fundamentals/src/content/get-started/origin-health/business.md [hunk 19]
```diff
@@ -1,5 +1,5 @@
 ---
-order: 1
+order: 2
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/origin-health/business.md]: I believe setting more pages to the same order number within one directory will result in them being ordered alphabetically — is this intentional?

File: products/fundamentals/src/content/get-started/network-ports.md [hunk 20]
```diff
@@ -0,0 +1,89 @@
...
+---
+
+# Network ports
+
+Learn which network ports Cloudflare proxies by default and how to enable Cloudflare's proxy for additional ports.
+
+## Network ports compatible with Cloudflare's proxy
+
+Cloudflare, by default, proxies traffic destined for the HTTP/HTTPS ports listed below.
+
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/network-ports.md]: extra line

File: products/fundamentals/src/content/get-started/http-request-headers.md [hunk 21]
```diff
@@ -0,0 +1,106 @@
...
+</div>
+</details>
+
+<details>
+<summary>CF-Worker</summary>
+<div>
+
+The CF-Worker request header is added to an edge Worker subrequest that identifies the host that spawned the subrequest. This is useful when you want to protect yourself against cross-zone worker subrequests. For example: ``CF-Worker: example.com``.
+
+You can add CF-Worker header on server logs similar to the way you add the [CF-RAY](https://support.cloudflare.com/hc/en-us/articles/203118044-Gathering-information-for-troubleshooting-sites#h_f7a7396f-ec41-4c52-abf5-a110cadaca7c) header. Just add ``$http_cf_worker`` in the log format file: ``log_format cf_custom "CF-Worker:$http_cf_worker"'``
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/http-request-headers.md]:
```suggestion
You can add CF-Worker header on server logs similar to the way you add the [CF-RAY](https://support.cloudflare.com/hc/en-us/articles/203118044-Gathering-information-for-troubleshooting-sites#h_f7a7396f-ec41-4c52-abf5-a110cadaca7c) header. To do that, add ``$http_cf_worker`` in the log format file: ``log_format cf_custom "CF-Worker:$http_cf_worker"'``
```

File: products/fundamentals/src/content/get-started/network-ports.md [hunk 22]
```diff
@@ -0,0 +1,89 @@
...
+pcx-content-type: concept
+---
+
+# Network ports
+
+Learn which network ports Cloudflare proxies by default and how to enable Cloudflare's proxy for additional ports.
+
+## Network ports compatible with Cloudflare's proxy
+
+Cloudflare, by default, proxies traffic destined for the HTTP/HTTPS ports listed below.
```
Comment (abracchi-tw) [inline products/fundamentals/src/content/get-started/network-ports.md]:
```suggestion
By default, Cloudflare proxies traffic destined for the HTTP/HTTPS ports listed below.
```

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md (hunk 2 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/cloudflare-cookies.md]: Good idea. I used the backticks.

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md (hunk 3 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/cloudflare-cookies.md]: Done.

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md (hunk 4 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/cloudflare-cookies.md]: I couldn't find this information in the documentation.

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md (hunk 5 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/cloudflare-cookies.md]: Agree.

**Review by abracchi-tw — APPROVED** [review]

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md (hunk 7 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/cloudflare-cookies.md]: I see. I think I would remove it: A separate \__cf_bm cookie is generated for each site that an end user visits, as Cloudflare does not track users from site to site or from session to session. The \__cf_bm cookie is generated independently by Cloudflare, and does not correspond to any user ID or other identifiers in a customer’s web application.

File: products/fundamentals/src/content/get-started/cloudflare-cookies.md (hunk 8 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/cloudflare-cookies.md]: done

File: products/fundamentals/src/content/get-started/how-cloudflare-works.md (hunk 9 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/how-cloudflare-works.md]: I see, it's dubious. I think it's the second one: More than just using Cloudflare's [Content Delivery (CDN)](https://www.cloudflare.com/learning/cdn/what-is-a-cdn/) services, customers rely on Cloudflare’s global network to enhance security, performance, and reliability of anything connected to the Internet.

File: products/fundamentals/src/content/get-started/how-cloudflare-works.md (hunk 14 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/how-cloudflare-works.md]: That's right. I would remove that parentheses. Note that additional terms may apply to:

- Self-serve domains with add-on features
- Domains in the Enterprise plan (contact your Cloudflare Account Team for additional details)

File: products/fundamentals/src/content/get-started/how-cloudflare-works.md (hunk 15 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/how-cloudflare-works.md]: done

File: products/fundamentals/src/content/get-started/http-request-headers.md (hunk 17 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/http-request-headers.md]: <summary>CF-IPCountry</summary> This one?

File: products/fundamentals/src/content/get-started/http-request-headers.md (hunk 18 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/http-request-headers.md]: yeah, I used it because it was code.

File: products/fundamentals/src/content/get-started/origin-health/business.md (hunk 19 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/origin-health/business.md]: Yeah, I wanted the the page Protect your origin server to appear in a second position and How Cloudflare works in the first position.

File: products/fundamentals/src/content/get-started/network-ports.md (hunk 20 above)
Comment (angelampcosta) [reply products/fundamentals/src/content/get-started/network-ports.md]: done

File: products/fundamentals/src/content/get-started/http-request-headers.md (hunk 18 above)
Comment (pedrosousa) [reply products/fundamentals/src/content/get-started/http-request-headers.md]: You need a single back tick on each side: `` `X-Forwarded-For: 203.0.113.1` ``. We don't use double back ticks.
